subprojects/core-api/src/main/java/org/gradle/api/file/CopySpec.java

```diff
@@ -135,7 +135,7 @@ public interface CopySpec extends CopySourceSpec, CopyProcessingSpec, PatternFil
 
     /**
      * The strategy to use when trying to copy more than one file to the same destination. Set to {@link DuplicatesStrategy#INHERIT}, the default strategy, to use
-     * the strategy inherited from the parent copy spec, if any, or {@link DuplicatesStrategy#INCLUDE} if this copy spec has no parent.
+     * the strategy inherited from the parent copy spec, if any, or {@link DuplicatesStrategy#FAIL} if this copy spec has no parent.
```

Comment:
❓ From a quick look, I'm not 100% sure this is correct: In https://github.com/gradle/gradle/blob/e1568ff83ddecf64248260376da957b986d625e7/platforms/core-configuration/file-operations/src/main/java/org/gradle/api/internal/file/copy/DefaultCopySpec.java#L774 the default is still `INCLUDE` however in https://github.com/gradle/gradle/commit/a1de6d03c5666d98f17c653ab8cf9771a32e53b2#diff-d4b99218127787f457d41d4b916629ef09f5d1b2a5dc809a505b6a1326b95528R53 the behaviour changes if there is no explicit configuration.

@ov7a Is it valid to claim that the default of `CopySpec` is `FAIL`, or should we document this facts at the tasks where the relevant codepath is executed, or should we also change the default value in `DefaultCopySpec`?

Reply:
I think this part javadoc should be rephrased to something like:
```
If no explicit deduplication strategy is set, but duplicates are found, an error is thrown
```

Reply:
Added a clear statement in the description. Let me know if it is acceptable.

Reply:
I think this part
```
* If the parent copy spec does not have a strategy, the default is {@link DuplicatesStrategy#FAIL}.
```
Should be removed completely. The actual failure happens in a different place and results in a different exception. So the default does not matter that much.

Reply:
Fixed.

Reply:
Now it says:
>      * The strategy to use when trying to copy more than one file to the same destination.
>      * Defaults to {@link DuplicatesStrategy#INHERIT}, the strategy inherited from the parent copy spec.
>      * If no explicit deduplication strategy is set, but duplicates are found, an error is thrown.

Reply:
LGTM